packages/console/src/pages/UserDetails/components/UserLogs.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import AuditLogTable from '@/components/AuditLogTable';
+
+import * as styles from '../index.module.scss';
+
+type Props = {
+  userId: string;
+};
+
+const UserLogs = ({ userId }: Props) => {
+  return (
+    <div className={styles.logs}>
```

Comment:
Just wondering, could we remove this wrapper?  apply the logs className to the `<AuditLogTable />`

Reply:
Currently, this `<AuditLogTable />` is wrapped with `React.Fragment`, and if we want to apply styles to this component, we need to wrap it with `<div>`, which will break the layout on the `AuditLog` page.
Maybe we can add an issue If we need to refactor this component.

Reply:
Is OK to wrap table with `<div>`